react/finger-exercises/redux/02 - react-redux/src/redux/cart/reducer.js

```diff
@@ -0,0 +1,46 @@
+import { actions } from './actions';
+
+const initialState = {
+  bookSelected: [],
+  open: false
+};
+
+// i don't like the sintax of this func but is the only way that the linter don't bother
+const incrementById = (arr, id) =>
+  arr.map(elem => {
+    if (elem.id === id) {
+      const aux = { ...elem };
+      aux.quantity += 1;
+      return aux;
+    }
+    return elem;
```

Comment:
What do you think to write it like arr.map(elem => elem.id === id ? {...elem, quantity: elem.quantity + 1 } : elem)?